app/main.py

```diff
@@ -2,0 +1,23 @@
+from os import mkdir, replace
+import os
+
+
+def move_file(command):
+    parts = command.split()
+    if len(parts) != 3 or parts[0] != "mv":
+        raise ValueError("Invalid command format. Use: mv <source> <destination>")
+
+    source = parts[1]
+    destination = parts[2]
+
+    three = destination.split("/")
+    array = []
+    for i in range(0, len(three)-1):
+        array.append(three[i])
+        if not os.path.exists("/".join(array)):
+            mkdir("/".join(array))
```

Comment:
The logic for creating directories is correct, but it can be simplified. Instead of manually splitting the path and creating directories one by one, you can use `os.makedirs(destination, exist_ok=True)` which will create all necessary directories in the path if they do not exist.